public/index.html

```diff
@@ -12,9 +12,11 @@
     <h1>Xem Phim Cùng Nhau</h1>
 
     <div id="roomSelection">
+        <label for="videoInput">Local Video:</label>
+        <input type="file" id="videoInput" accept="video/*" placeholder="Sử dụng local video"/>
```

Comment:
i think we only need mp4 file. So it should be
```suggestion
        <input type="file" id="videoInput" accept="video/*" placeholder="Sử dụng local video"/>
        <input type="file" id="videoInput" accept="video/mp4" placeholder="Sử dụng local video"/>
```